.PyFunceble.yaml

```diff
@@ -3,7 +3,7 @@ aggressive: false
 auto_continue: true
 ci: true
 ci_autosave_commit: '[Autosave][Dead-Hosts::repository-structure]'
-ci_autosave_final_commit: '[Final/Result][Dead-Hosts::repository-structure]'
+ci_autosave_final_commit: '[Final/Result] Dead-Hosts: AdBlockID Plus'
```

Comment:
Is something not working @realodix ? Should I look further into it ?

Reply:
How change the commit message? https://github.com/dead-hosts/AdBlockID_git_realodix/pull/4 it looks not working.

Reply:
Apparently it is working: https://github.com/dead-hosts/AdBlockID_git_realodix/blob/9b9646ca3e7991ad62f66e77e0715f2bef1cc9bd/info.json#L5